### ISSUE TITLE
Error acquiring permit: null.

### ISSUE THREAD

**Issue by ddebrunner** [IBMStreams/streamsx.messagehub#38]
Saw this running `MessageHubConsumer`  (1.2.3) in a consistent region after/during a PE restart.

```
Errors have been detected in your application(s). For more information, see the Application Errors card and the Log Viewer.
Job[84], PE[99], Message[CDIST2156E]:
Error acquiring permit: null.
```

The region recovered successfully but the console continued to show this as an error.

Maybe if this is a transient error due to restart it should not be logging it as a real error???

**Comment by ddebrunner** [comment]
```
12 Feb 2018 23:47:45.692 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[AbstractKafkaConsumerOperator.java:com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator$1.run:416]  - Operator error
12 Feb 2018 23:47:45.695 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - java.lang.InterruptedException
12 Feb 2018 23:47:45.695 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1313)
12 Feb 2018 23:47:45.695 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at java.util.concurrent.CountDownLatch.await(CountDownLatch.java:242)
12 Feb 2018 23:47:45.696 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at com.ibm.streamsx.kafka.clients.consumer.KafkaConsumerClient.sendStopPollingEvent(KafkaConsumerClient.java:476)
12 Feb 2018 23:47:45.696 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator.produceTuples(AbstractKafkaConsumerOperator.java:490)
12 Feb 2018 23:47:45.696 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator.access$000(AbstractKafkaConsumerOperator.java:46)
12 Feb 2018 23:47:45.697 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator$1.run(AbstractKafkaConsumerOperator.java:414)
12 Feb 2018 23:47:45.697 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at java.lang.Thread.run(Thread.java:811)
12 Feb 2018 23:47:45.697 [6062] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[?:?:0]  - at com.ibm.streams.operator.internal.runtime.OperatorThreadFactory$2.run(OperatorThreadFactory.java:137)
12 Feb 2018 23:48:16.100 [9607] ERROR #splapptrc,J[86],P[103],PartCR.G[0].Msgs M[AbstractKafkaConsumerOperator.java:com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator.produceTuples:459]  - CDIST2156E Error acquiring permit: null
```

**Comment by ddebrunner** [comment]
Looks like an `InterruptedException` is not being treated as shutdown.

**Comment by ddebrunner** [comment]
See this about 50% of restarts of the PE containing the operator.

**Comment by RolefH** [comment]
Must be corrected in Kafka core toolkit.

**Comment by RolefH** [comment]
corrected in v1.2.4
